src/cbc_sdk/platform/legacy_alerts.py

```diff
@@ -342,7 +342,8 @@ def set_time_range(self, key, **kwargs):
         Returns:
             AlertSearchQuery: This instance.
         """
```

Comment:
In order to support the new time range filter this may need to also support a signature on the Alert that matches the new functionality.

 v7 Call:

cb.select(HostBasedFirewallAlert).set_time_range(range="-6m") --> generates a request body like 

`{
    "time_range": {
        "range": "-6m"
    },
    "criteria": {
        "alert_type": [
            "HOST_BASED_FIREWALL"
        ]
    }
}`

v6 Call:
This is generated correctly.
cb.select(HostBasedFirewallAlert).set_time_range("create_time", range="-6m") --> generates a request body like 

`{
    "criteria": {
          "backend_timestamp": {"range": "-6m"}, 
           "type": ["HOST_BASED_FIREWALL"]
    }, 
      "query": "", 
    "rows": 1
}`


https://developer.carbonblack.com/reference/carbon-black-cloud/platform/latest/alerts-api/#time-range-filter